docs/cluster-configuration-v1beta1-api.md

```
@@ -227,11 +227,13 @@ Set the `decentralizedLiveMigration` feature gate to true in order to enable dec
 
 **Graduation Status**: Alpha
 
-### enableMultiArchBootImageImport Feature Gates
+### enableMultiArchBootImageImport Feature Gates (Deprecated)
 Set the `enableMultiArchBootImageImport` feature gate to true in order to enable the golden images support in
 heterogeneous clusters. See [Golden Images](#golden-images-in-heterogeneous-clusters) for more information.
 
-**Note**: this feature is in Developer Preview.
+**Note**: this feature is GA, and the feature gate is deprecated. Use the 
+`spec.workloadSources.enableMultiArchBootImageImport` field in the **`v1`** API version, instead. The `v1` API version 
+documentation is [here](./cluster-configuration.md#golden-images-in-heterogeneous-clusters)
```

markdownlint-cli2 (0.23.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)